### ISSUE TITLE
Horizontal scroll bar for plots in view.html

### ISSUE THREAD

**Issue by sanketd11** [att/rcloud#1349]
Similar to issue #1239 .
In view.html mode, panels cannot be hidden to view a stretched plot.
![plot cannot be seen](https://cloud.githubusercontent.com/assets/9251570/6243583/658c1ce4-b76a-11e4-8850-11401b7e0879.PNG)

**Comment by gordonwoodhull** [comment]
Yeah, I still don't want horizontal scrollbars.  I'd prefer to be able to change the width of the view.  Retitling

**Comment by gordonwoodhull** [comment]
Also, check what edit.html is doing about this - why does it work there?

**Comment by sanketd11** [comment]
In edit.html, the side panels can be resized to see the stretched graph. Here the worst case scenario can be for the dense graph, when it needs to be stretched even more after all side panels are closed to see the point values as shown below in screenshot 
when all side panels are closed the issue is same as that is replicated in view.html 

![plot](https://cloud.githubusercontent.com/assets/9251570/6262786/54c2939a-b830-11e4-9ca3-ce7eb8c5987e.PNG)
